packages/react-devtools-extensions/src/main.js

```diff
@@ -135,7 +135,7 @@ function createPanelIfReactLoaded() {
         // Initialize the backend only once the Store has been initialized.
         // Otherwise the Store may miss important initial tree op codes.
         chrome.devtools.inspectedWindow.eval(
-          `window.postMessage({ source: 'react-devtools-inject-backend' }, window.origin);`,
+          `window.postMessage({ source: 'react-devtools-inject-backend' }, '*');`,
```

Comment:
Should we use `window.origin || "*"` or will `"*"` be sufficient for our Trusted Types case too?

Reply:
That looks good to me too but "*" will also be sufficient

Reply:
Ok! Just wanted to verify. If "*" will cover both use cases, let's stick with that.